foreman/foreman.spec

```diff
@@ -31,6 +31,14 @@ Source7: %{name}-plugins.repo
 Source8: %{name}.gpg
 BuildArch:  noarch
 
+Requires: %{name}-core = %{version}-%{release}
+Requires: %{name}-debug
+Requires: %{?scl_prefix}rubygem(foreman-tasks) >= 0.8.5
```

Comment:
1. Should this be in a `%package tasks` like other optional functionality, e.g. -ec2?
1. Dependency won't resolve, packages in the main repo can't depend on the contents of the plugin repo.